DataFormats/Common/interface/DetSetVectorNew.h

```diff
@@ -133,23 +186,32 @@ namespace edmNew {
       typedef typename DetSetVector<T>::id_type id_type;
       typedef typename DetSetVector<T>::size_type size_type;
 
+      // here just to make the compiler happy
+      static DetSetVector<T>::Item & dummy() {
+        assert(false);
+	static  DetSetVector<T>::Item d; return d;
+      }
       FastFiller(DetSetVector<T> & iv, id_type id, bool isaveEmpty=false) : 
```

Comment:
Were all uses of ```FastFiller``` replaced with ```TSFastFiller```? If so, then in a subsequent pull request we should remove ```FastFiller```.